### PR TITLE
XW-2018 | animations for global navigation

### PR DESCRIPTION
**Pull request by Warkot**
## Links

* https://wikia-inc.atlassian.net/browse/XW-2018
* https://github.com/Wikia/design-system/pull/63
* https://github.com/Wikia/app/pull/11344

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: front/main/bower.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "Autolinker.js": "0.22.0",
     "cropper": "0.11.0",
-    "design-system": "Wikia/design-system#0.9.0",
+    "design-system": "Wikia/design-system#XW-2018",
```
Comment (tomasznapieralski) [inline front/main/bower.json]: remember to use DS version

File: front/main/app/templates/components/design-system/global-navigation.hbs [hunk 2]
```diff
@@ -10,26 +10,28 @@
...
-				<button class="wds-button wds-is-text wds-global-navigation__search-close" type="reset">
-					{{svg 'wds-icons-cross' class='wds-icon wds-icon-small wds-global-navigation__search-close-icon'}}
-				</button>
-			</div>
-			<button class="wds-button wds-global-navigation__search-submit" type="submit">
-				{{svg 'wds-icons-arrow' class='wds-icon wds-icon-small wds-global-navigation__search-submit-icon'}}
-			</button>
-		</form>
+		<div class="wds-global-navigation__links-and-search">
+			{{#if model.fandom_overview}}
```
Comment (hakubo) [inline front/main/app/templates/components/design-system/global-navigation.hbs]: we don't need this if here each will do that for us

Comment (Warkot) [inline front/main/app/templates/components/design-system/global-navigation.hbs]: We just added a wrapper, but I agree we can fix it. 😉  👍

Comment (Warkot) [inline front/main/app/templates/components/design-system/global-navigation.hbs]: Just to be sure, are we safe to access one step deeper (`model.fandom_overview.links`) without checking the middle level (`model.fandom_overview`)? Vanilla JS would rise an exception `Cannot read property 'links' of undefined`.

Comment (hakubo) [inline front/main/app/templates/components/design-system/global-navigation.hbs]: yup: https://ember-twiddle.com/f7eaf52aacf9ef0b7d8267f741c732bd?openFiles=templates.application.hbs%2C

**Pull request merged**